src/Raven.Server/Documents/Versioning/VersioningStorage.cs

```diff
@@ -136,6 +139,22 @@ public void PutVersion(DocumentsOperationContext context, string collectionName,
             var revisionsCount = IncrementCountOfRevisions(context, prefixSlice, 1);
             DeleteOldRevisions(context, table, prefixSlice, configuration.MaxRevisions, revisionsCount);
 
+            PutInternal(context, key, newEtagBigEndian, document, table);
+        }
+
+        /// <summary>
+        /// Should be used from smuggler import
+        /// </summary>
+        public void Put(DocumentsOperationContext context, string key, long etag, BlittableJsonReaderObject document)
```

Comment:
Bad naming, include the name and intent in the method name, instead of a comment.